.buildkite/pipelines/es_serverless/verify_es_serverless_image.yml

```diff
@@ -44,26 +44,43 @@ steps:
         timeout_in_minutes: 60
         retry:
           automatic:
-            - exit_status: '-1'
+            - exit_status: "-1"
               limit: 3
 
       - label: "Pick Test Group Run Order (FTR + Integration)"
         command: .buildkite/scripts/steps/test/pick_test_group_run_order.sh
         depends_on: build
         timeout_in_minutes: 10
         env:
-          FTR_CONFIGS_SCRIPT: 'TEST_ES_SERVERLESS_IMAGE=$ES_SERVERLESS_IMAGE .buildkite/scripts/steps/test/ftr_configs.sh'
-          JEST_INTEGRATION_SCRIPT: 'TEST_ES_SERVERLESS_IMAGE=$ES_SERVERLESS_IMAGE .buildkite/scripts/steps/test/jest_integration.sh'
-          FTR_CONFIG_PATTERNS: '**/test_serverless/**'
-          FTR_EXTRA_ARGS: '$FTR_EXTRA_ARGS'
-          LIMIT_CONFIG_TYPE: 'functional,integration'
+          FTR_CONFIGS_SCRIPT: "TEST_ES_SERVERLESS_IMAGE=$ES_SERVERLESS_IMAGE .buildkite/scripts/steps/test/ftr_configs.sh"
+          JEST_INTEGRATION_SCRIPT: "TEST_ES_SERVERLESS_IMAGE=$ES_SERVERLESS_IMAGE .buildkite/scripts/steps/test/jest_integration.sh"
+          FTR_CONFIG_PATTERNS: "**/test_serverless/**"
+          FTR_EXTRA_ARGS: "$FTR_EXTRA_ARGS"
+          LIMIT_CONFIG_TYPE: "functional,integration"
         retry:
           automatic:
-            - exit_status: '*'
+            - exit_status: "*"
+              limit: 1
+
+      # Security tests step is separated from the above one only for clarity, as the way they execute is different (folder and config pattern) than
+      # the rest of the tests. An alternative to group them would be to add the config pattern as a comma separatated list in the above step.
+      - label: "Pick Test Group Run Order (FTR + Integration) For Security"
+        command: .buildkite/scripts/steps/test/pick_test_group_run_order.sh
```

Comment:
We cannot run this script twice without major alterations. It uploads artifacts and meta data to Buildkite with specific names/keys which are then used by the individual FTR steps.

Reply:
which means that we need the patter comma separated list. All good. Doing it now.